projects/cds/src/services/cds-endpoints.service.ts

```diff
@@ -0,0 +1,55 @@
+import { Injectable } from '@angular/core';
+import { DynamicTemplate } from '../../../core/src/config/utils/dynamic-template';
+import { CdsConfig } from '../cds-config';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CdsEndpointsService {
```

Comment:
I will get back to you about this one. Is there a reason why you couldn't re-use our OccEndpointsService?

Reply:
Only because these endpoints aren't within the storefront, but are from the CDS cloud, so not really OCC related REST endpoints. If this doesn't matter, we can obviously change this

Reply:
Got it. Pity.
Let's continue in the comment above.